src/h_vialib/client.py

```diff
@@ -29,14 +29,15 @@ def is_pdf(self):
 class ViaClient:  # pylint: disable=too-few-public-methods
     """A small wrapper to make calling Via easier."""
 
-    def __init__(self, service_url, host_url):
+    def __init__(self, service_url, host_url, secure_token):
         """Initialize a ViaClient pointing to a `via_url` via server.
 
         ￼
         :param via_url location of the via server
         :param host_url origin of the request
         """
         self.service_url = urlparse(service_url)
+        self._secure_token = secure_token
```

Comment:
We should probably use the `ViaSecureURLToken` here directly so the client doesn't have to wire together lots of different classes. The only thing we can't already know up front is the secret.

Reply:
Yeah I think you are right, here doesn't look too bad but on LMS it weird to have to import and construct a ViaSecureURLToken